apps/web/components/apps/AppPage.tsx

```diff
@@ -225,7 +225,6 @@ export const AppPage = ({
                         multiInstall
                         concurrentMeetings={concurrentMeetings}
                         paid={paid}
-                        dirName={dirName}
```

Comment:
If we're removing the use of the `dirName`, let's remove the prop from the component as well.